src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -24,6 +24,12 @@ You can configure the `ZipkinExporter` through
    associated with the telemetry has "service.name" defined, then it'll be
    preferred over this option.
 * `Endpoint`: URI address to receive telemetry (default `http://localhost:9411/api/v2/spans`).
```

Comment:
Although not added by this PR, I wonder if we require the endpoint to have a training `/` or not or both should work.

Reply:
I think both should work. Example here https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=net-5.0#examples shows both without actually specifying recommended way. In code we do not use trailing '/' so we can keep it same here.